lib/google_sign_in/engine.rb

```diff
@@ -1,15 +1,26 @@
 require 'rails/engine'
+require 'google_sign_in' unless defined?(GoogleSignIn)
```

Comment:
Account for `bin/rails` requiring ENGINE_PATH directly, before `google_sign_in` itself.